client_code/kompot/_persist.py

```diff
@@ -0,0 +1,248 @@
+# SPDX-License-Identifier: MIT
+# Copyright (c) 2021 anvilistas
+
+"""A module to provide persistence mechanisms for a class.
+
+e.g.
+
+For a class named 'Book', we might have a data table named 'book' where each row
+corresponds to instance of the Book class. We might also have server functions to
+add new row or to update existing entries:
+
+@row_backed_class
+class Book:
+    server_functions = {
+        "create": "add_book",
+        "get": "get_book",
+        "update": "update_book",
+        "delete": "delete_book",
+    }
+
+Create a new Book instance and fetch its row from the db:
+
+book = Book()
+book.get(title="Fluent Python")
+
+This assumes a server function exists named 'get_book', which take a title argument
+and returns a data tables row.
+
+All the columns from the row will now be available as attributes on the Book instance.
+e.g.
+
+print(book.title)
+>>> Fluent Python
+"""
+
+import anvil.server
+
+from ._register import register
+from ._rpc import call
+
+__version__ = "0.0.1"
+
+
+@register
+@anvil.server.portable_class
+class Despatcher:
```

Comment:
I guess it's private so not a big deal that it's `Despatcher` but I think `Dispatcher` would probably be better.